python/dex/api.py

```diff
@@ -7,11 +7,10 @@
 import ctypes
 import pathlib
 import atexit
+from pkg_resources import resource_filename
 from typing import List
 
-here = pathlib.Path(__file__).parent.absolute()
-
-lib = ctypes.cdll.LoadLibrary(here / 'libDex.so')
+lib = ctypes.cdll.LoadLibrary(resource_filename('dex', 'libDex.so'))
```

Comment:
I do wonder if we need to use `pkg_resources` here, given that we really want to search within the package. A relative path might still be good enough, but I guess this is still more robust anyway 🤷‍♂️

Reply:
seems that pkg_resources is a safer approach

https://stackoverflow.com/questions/39104/finding-a-file-in-a-python-module-distribution
http://peak.telecommunity.com/DevCenter/PythonEggs#accessing-package-resources

```
zhangqiaorjc@qiao-gpu-1:~$ tree .local/lib/python3.8/site-packages/dex/
.local/lib/python3.8/site-packages/dex/
├── __init__.py
├── __pycache__
│   ├── __init__.cpython-38.pyc
│   ├── api.cpython-38.pyc
│   └── native_function.cpython-38.pyc
├── api.py
├── interop
│   ├── __init__.py
│   ├── __pycache__
│   │   ├── __init__.cpython-38.pyc
│   │   └── jax.cpython-38.pyc
│   └── jax.py
├── libDex.so
└── native_function.py
```

Reply:
Cool, let's keep that!